### PR TITLE
Mark active node with a halo

### PR DESCRIPTION
**Pull request by philippotto**
Highlight the active node via a halo (in orthogonal mode). In the arbitrary modes, the active node is centered most of the time, anyway. Additionally, it is still round.

### URL of deployed dev instance (used for testing):
- https://iterateactivenode.webknossos.xyz

### Steps to test:
- switch between view modes and change active node

### Issues:
- see #3628 

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [X] Ready for review

### CODE REVIEW

File: app/assets/javascripts/oxalis/geometries/materials/node_shader.js [hunk 1]
```diff
@@ -247,17 +270,23 @@ void main()
...
         delta = fwidth(r);
-        alpha = 1.0 - smoothstep(1.0 - delta, 1.0 + delta, r);
+        alphaOuter = 1.0 - smoothstep(0.0, delta, abs(1.0 - delta - r));
+        alphaInner = 1.0 - smoothstep(1.0 - delta, 1.0 + delta, r / (v_innerPointSize / v_outerPointSize));
+        alphaOuter = max(0.0, alphaOuter - alphaInner);
       #endif
 
-      gl_FragColor = vec4(color, alpha);
+      vec4 inner = vec4(color, alphaInner);
+      // Make the outer ring white
```
Comment (daniel-wer) [inline app/assets/javascripts/oxalis/geometries/materials/node_shader.js]: Outdated, the outer ring has the same color as the inner circle, can probably simply be removed.

**Comment by jstriebel** [comment]
Nice :sparkles: :red_circle: :sparkles: 

Could you add a short gif to the changelog, if this makes it in?

**Comment by philippotto** [comment]
> Could you add a short gif to the changelog, if this makes it in?

Sorry, I tried to create a gif, but somehow the recording is so choppy, that the result is unusable..

**Pull request merged**